Changes

```diff
@@ -1,5 +1,10 @@
 Changes file for Date-Holidays
 
+1.04 2017-05-30 Feature release, update not required
```

Comment:
I just wanted to say that this date will always be inaccurate if the pull request are waiting for long enough